Zend/zend_errors.h

```diff
@@ -20,9 +20,8 @@
 #ifndef ZEND_ERRORS_H
 #define ZEND_ERRORS_H
 
-#define E_ERROR				(1<<0L)
```

Comment:
Yeah, it was my guess as well that it would only matter if the values are greater than 32 bits, but wanted to make sure that I don't miss anything else. Thanks for the rest of the explanation as well!